demo/python/jupyter/e2e-demo.ipynb

```diff
@@ -33,7 +33,7 @@
     "import securexgboost as xgb\n",
     "import os\n",
     "\n",
-    "HOME_DIR = os.path.abspath('') + \"/../../../\"\n",
+    "HOME_DIR = os.getcwd() + \"/../../../\"\n",
```

Comment:
Please revert this file except for any API changes

Reply:
done